src/main/java/Pages/ConfigPage.java

```diff
@@ -0,0 +1,36 @@
+package Pages;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+import static com.codeborne.selenide.Selenide.*;
+
+public class ConfigPage {
+
+    String configPageCSS = "div.row.page-tit > h2";
+    String configTitleFieldCSS = "#configsectionmodel-main___site__title";
+    String configSubmitButtonCSS = "a.btn.btn-green.btn-submit";
+    public static String configTitleFieldTest = "autotitle";
+    public static String configTitleFieldDefault = "Название сайта";
+
+    public SelenideElement getConfigPageMarker() {
+        return $(By.cssSelector(configPageCSS));
+    }
+
+    public static String getConfigTitleFieldTest() {
+        return configTitleFieldTest;
+    }
+
+    public static String getConfigTitleFieldDefault() {
+        return configTitleFieldDefault;
```

Comment:
totally don't understand these methods. why do we need both? maybe "autotitle" and "Название сайта" has to be passed as parameters? instead of doing it through separate locators